src/classes/data1dstore.cpp

```diff
@@ -28,18 +28,19 @@ bool Data1DStore::addData(std::string_view dataName, LineParser &parser, int sta
 // Check to see if the named data is present in the store
 bool Data1DStore::containsData(std::string_view name) const
 {
-    return std::find_if(data_.begin(), data_.end(), [&name](auto &data) { return data.first.tag() == name; }) != data_.end();
+    return std::find_if(data_.begin()->get(), data_.end()->get(), [&name](auto &data) { return data.first.tag() == name; }) !=
+           data_.end()->get();
```

Comment:
This doesn't look right and wouldn't traverse the vector:
```suggestion
    return std::find_if(data_.begin(), data_.end(), [&name](auto &data) { return data->first.tag() == name; }) !=
           data_.end();
```